### ISSUE TITLE
Confusing rating bar

### ISSUE THREAD

**Issue by alanshaw** [tableflip/sandstorm-appstore#179]
![screen shot 2015-06-22 at 15 33 40](https://cloud.githubusercontent.com/assets/152863/8284303/2874f0b0-18f4-11e5-980f-61ffd17f4600.png)

Are the icons the wrong way round?

Is a progress bar the right UI element for this kind of thing?

**Comment by jadeqwang** [comment]
I think this is improved from before, but can still be a bit confusing:

![image](https://cloud.githubusercontent.com/assets/1584904/8710235/274df450-2afd-11e5-9f64-12829a9384a7.png)

When I click :+1: , it's true, the buttons shown are the 3 actions I can take, but somehow, it _feels_ like I've set the state to :-1:, whereas it's merely that I have the option of clicking :-1: if I fat-fingered the :+1: .

I believe the answer to that is a toggle-style button for the [:+1:|:-1: ], where the :+1: doesn't disappear.

**Comment by neynah** [comment]
@jadeqwang I agree. I've suggested improvements on #189 but no response there yet.

**Comment by neynah** [comment]
@jadeqwang We've agreed on the revision of the toggle & buttons(let me know if there are any issues with these) but still waiting to hear back on the detailed breakdown of reviews.

**Comment by jadeqwang** [comment]
Yeah, this should probably get merged with #189
